standard/expressions.md

```diff
@@ -4793,7 +4792,7 @@ It is possible for anonymous function delegates to share some captured variables
 
 > *Example*: For example, if `F` is changed to
 >
-> <!-- Example: {template:"standalone-console", name:"InstantiationOfLocalVariables6", IgnoredWarnings:["CS8321"], additionalFiles:["DvoidNoArgs.cs"]} -->
+> <!-- Example: {template:"code-in-partial-class", name:"InstantiationOfLocalVariables6", IgnoredWarnings:["CS8321"], additionalFiles:["Caller.cs"], inferOutput: true} -->
```

Comment:
We *may* want to rename this to console-partial-class at some point to make it clear what the output type is. Although if I implement project properties in metadata, we may find we can reduce the number of templates significantly anyway. Not a problem for now.

Reply:
OK